### PR TITLE
enable NFS support

### PR DESCRIPTION
**Pull request by ganeshmaharaj**
Signed-off-by: Ganesh Maharaj Mahalingam <ganesh.mahalingam@intel.com>

### CODE REVIEW

File: lib/vagrant-guests-clearlinux/cap/nfs_client.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    module Cap
+      class NFS
+        def self.nfs_client_installed(machine)
+          machine.communicate.test("test -x /usr/bin/mount.nfs")
+        end
+
+        def self.nfs_client_install(machine)
+          comm = machine.communicate
+          comm.sudo([
+            "swupd bundle-add storage-utils",
```
Comment (AntonioMeireles) [inline lib/vagrant-guests-clearlinux/cap/nfs_client.rb:12]: invoking `swupd bundle-add storage-utils` here is redundant as the `storage-utils` bundle is **always** available unconditionally, by Default, in the stock ClearLinux vagrant boxes we publish.

**Comment by AntonioMeireles** [comment]
@ganeshmaharaj 

Hi, **MANY**, **MANY** thanks for this ...

i've added already some comments... won' t be able probably to test / merge before next week as i'm in the _road_ (in Vacouver at SURICON) 


Best Regards, 

António

**Comment by ganeshmaharaj** [comment]
@AntonioMeireles thanks. Take your time on the merge. I was not able to spot the comments youi had added on the patch. Will check for it next week once you are back from your travel.

**Review by AntonioMeireles — CHANGES_REQUESTED** [review]
@ganeshmaharaj 

(back in Portugal :-) ) clarified my comments. 

basically it's just the `"swupd bundle-add storage-utils",` that is superfluous. 

Other than that, and at sight all will looks fine. 

thanks again!

File: lib/vagrant-guests-clearlinux/cap/nfs_client.rb (hunk 1 above)
Comment (ganeshmaharaj) [reply lib/vagrant-guests-clearlinux/cap/nfs_client.rb:12]: @AntonioMeireles the kvm images did not have those when i checked the last time. Are you adding them as part of your packaging or does the vmdk images have them?

**Comment by ganeshmaharaj** [comment]
@AntonioMeireles should give you some background along with this. I realized that we didn't have a libvirt image for clearlinux and i started creating them https://app.vagrantup.com/gmmaha/boxes/clearlinux

**Comment by AntonioMeireles** [comment]
ah... do you have the source available for those?... would make sense to
just add libvirt support to my pipeline (was on my todo list already).
Patches welcome ;-). in the meantime, given that, i'll merge your patch as
is (when near my laptop). plz do note that the guest plugin makes a lots of
assumptions about how the boxes are built (plz see my packer tooling) that
you should follow, otherwise you may get some undesired "features".

all the best,


António

**Comment by ganeshmaharaj** [comment]
@AntonioMeireles absolutely. https://github.com/ganeshmaharaj/clearlinux/blob/master/libvirt-box/gen_vagrant_libvirt.sh that is the simple script that builds the libvirt box for me. Yes, i was going to look at your packer tooling and get libvirt going, but time crunch pushed me to a shell script. I will definitely work on moving it into your tooling. The guest plugin does work perfectly with the KVM image as of today (The couple of use-cases i have tested it).

**Comment by AntonioMeireles** [comment]
@ganeshmaharaj it's in ... v1.0.14 is out and pushed to rubygems.org. 

please report back if anything odd arises. 

**THANKS**!

**Comment by ganeshmaharaj** [comment]
@AntonioMeireles thank you so much. I will definitely let you know.

**Pull request merged**